R/plumber.R

```diff
@@ -264,7 +263,7 @@ Plumber <- R6Class(
           callback = swaggerCallback,
           quiet = quiet
         )
-        on.exit(unmount_docs(self, docs_info), add = TRUE)
+        #on.exit(unmount_docs(self, docs_info), add = TRUE)
```

Comment:
I believe this still needs to be there as the docs will want to be mounted again on a followup run.